client/ayon_blender/api/pipeline.py

```diff
@@ -646,10 +654,21 @@ def ls() -> Iterator:
             continue
 
         for shader_node in material_node_tree.nodes:
-            if not shader_node.get(AVALON_PROPERTY):
-                continue
-
-            if shader_node.get(AVALON_PROPERTY).get("id") not in container_ids:
+            ayon_shader_node = shader_node.get(
+                AYON_PROPERTY)
+            if not ayon_shader_node:
+                avalon_shader_node = shader_node.get(
+                    AVALON_PROPERTY)
+                if not avalon_shader_node:
+                    continue
+                else:
+                    ayon_shader_node[AYON_PROPERTY] = (
+                        avalon_shader_node
+                    )
+                    ayon_shader_node = avalon_shader_node
+                    del ayon_shader_node[AVALON_PROPERTY]
```

Comment:
No need for else if there is `continue`.
```suggestion
                ayon_shader_node[AYON_PROPERTY] = (
                    avalon_shader_node
                )
                ayon_shader_node = avalon_shader_node
                del ayon_shader_node[AVALON_PROPERTY]
```